src/lib/services/data-encoder.ts

```diff
@@ -7,34 +7,60 @@ import { stringifyWithBigInt } from '$lib/utilities/parse-with-big-int';
 
 import type { Payloads } from '$types';
 
-export async function convertPayloadsWithCodec({
-  payloads,
-  namespace,
-  settings,
-  accessToken,
-}: {
-  payloads: Payloads;
-  namespace: string;
-  settings: Settings;
-  accessToken: string;
-}): Promise<Payloads> {
+export function setCodecServerHeaders(
+  namespace: string,
+  settings: Settings,
+  accessToken: string,
+) {
+  let validationFailed = false;
   const endpoint = settings?.codec?.endpoint;
   const passAccessToken = settings?.codec?.passAccessToken;
 
   const headers = {
     'Content-Type': 'application/json',
     'X-Namespace': namespace,
+    credentials: 'same-origin',
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS

I think if this is set at all it preflights the request which was the breaking change last time so we should triple check that setting this doesn’t break everything again

Reply:
Aight ill remove this